assets/css/style.css

```diff
@@ -0,0 +1,86 @@
+h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #227452;
+}
```

Comment:
- [ ] Kindly delete the extra space between the title and subtitle to look similar to the Figma template. 💡 **Hint:** use margin-bottom and margin-top in the corresponding element.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/212429507-20e9cb19-d524-48ff-806e-a80d622d11de.png) | ![image](https://user-images.githubusercontent.com/58642949/212429583-a3b4e3e3-c1b3-404d-818a-a1e10fec456f.png) |